java/pom.xml

```diff
@@ -378,6 +341,64 @@
   </build>
 
   <profiles>
+    <!-- START FEATURE PROFILES -->
```

Comment:
Needs all the profiles added, not just these 4

Reply:
as agreeed no profile will be added for messaging feature